server/conf/application.dev.conf

```diff
@@ -62,5 +62,5 @@ maximum_session_duration_minutes = 7200
 api_generated_docs_enabled = true
 program_filtering_enabled = true
 name_suffix_dropdown_enabled = true
-fastforward_enabled = true
+customized_eligibility_message_enabled = true
```

Comment:
question (non-blocking): Is enabled customized eligibility message in dev an intended change?

Reply:
No, Merge conflict. That was move out of that files. I'll fix it. Good catch